recipes-ni/ni-systemformat/ni-systemformat.bb

```diff
@@ -42,4 +42,14 @@ inherit update-rc.d
 INITSCRIPT_NAME = "nitargetinfo"
 INITSCRIPT_PARAMS = "start 20 S ."
 
-RDEPENDS:${PN} += "bash"
+RDEPENDS:${PN} += "\
+	bash \
+	niacctbase \
+"
+# nisystemformat rdeps
+RDEPENDS:${PN} += "\
+	coreutils \
+	ni-netcfgutil \
+	util-linux-lsblk \
+"
+RDEPENDS:${PN}:append:x64 = " e2fsprogs-mke2fs"
```

Comment:
Why a separate x64 list here?

Reply:
I thought we were removing x64-specific code with the assumption that we support this image only on x64. Is that not the case here?

Reply:
I originally didn't expect that `e2fsprogs-mke2fs` would exist in the ARM feeds, but I suppose we don't build those in hardknott anyway, so it doesn't matter. Will fix.

Reply:
Fixed in https://github.com/ni/meta-nilrt/pull/483/commits/57a560de271c9ac09ee990368e98a9080279226c.